src/lib/systemd_journal/query_presenter.rb

```diff
@@ -82,12 +83,16 @@ def interval_description
         _("Since system's boot")
       when "-1"
         _("From previous boot")
-      else
+      when Hash
         dates = {
           since: Yast::Builtins.strftime(interval[:since], TIME_FORMAT),
           until: Yast::Builtins.strftime(interval[:until], TIME_FORMAT)
         }
         _("Between %{since} and %{until}") % dates
+      when nil
```

Comment:
The possibility of passing nil if not so clear in the yardoc documentation of `SystemdJournal::Query#initialize`. We should likely take the opportunity to improve it.

Reply:
@ancorgs well, issue is that it is default value. So it is a bit strange situation.